migrate/live-migration/index.md

```diff
@@ -33,6 +33,9 @@ Live migration works well when:
   inserts are batched. If your application exceeds this, you should use
   the [dual-write and backfill] migration method.
 
+Live migration has the following shortcomings:
```

Comment:
Should we also add about generated columns?

Reply:
Working on it in a separate change.